frontend/app/profile/page.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { getServerSession } from 'next-auth/next'
+import { authOptions } from '../api/auth/[...nextauth]/route'
+
+import { UserProfile } from '../../components/pages/UserProfile'
+import Error from '../error'
+import { getUser } from 'app/api'
+
+export default async function UserProfilePage() {
+  const session = await getServerSession(authOptions)
+  const userId = session?.user.id
+  const user = await getUser(userId)
```

Comment:
userId一度しか使わないのであれば

```suggestion
  const user = await getUser(session?.user.id)
```

でも良いかも！

Reply:
なるほど、勉強になります！